hooks/useTransactionExport.ts

```
@@ -0,0 +1,285 @@
+
+    // Clean up temp file after a delay (sharing might be async)
+    setTimeout(async () => {
+      try {
+        await FileSystem.deleteAsync(tempPath, { idempotent: true });
+      } catch (error) {
+        console.log('Failed to clean up temp file:', error);
+      }
+    }, 5000);
+  };
+
+  const exportToCSV = async (data: Record<string, unknown>[], filename: string): Promise<void> => {
```

ESLint

[error] 'exportToCSV' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)